Vagrantfile

```diff
@@ -42,7 +42,7 @@ Vagrant.configure(2) do |config|
     # ansible.verbose = 'vvv'
 
     ansible.groups = {
-      "figgy" => ["default"]
+      "vagrant" => ["default"]
```

Comment:
Not need but we probably want to make work differently

Reply:
Not sure what you want here.

Reply:
leave it as in. No change needed to the file.